CONSIDER/.htaccess

```diff
@@ -0,0 +1,3 @@
+Options +FollowSymLinks
+RewriteEngine on
+RewriteRule ^$ https://github.com/lhncbc/CDE/tree/master/CONSIDER [R=301]
```

Comment:
This is going to the github interface directly, is that what you wanted?

Reply:
Yes, we wanted that.. We hope to move the stuff later to either our own website or to a website maintained by National Library of Medicine. But for now, that is the best target URL and what we intended.